### ISSUE TITLE
Data loss when moving content using Filer to heading with trailing space

### ISSUE THREAD

**Issue by edgauthier** [NotePlan/plugins#353]
**Describe the bug**
When attempting to move content using the Filer plugin, and the target heading has a trailing space, the content is removed from the source note, but not added to the destination note. 

**To Reproduce**
1. Start with content in a source note (e.g. a daily note), and a target note with a heading (H3 in my example) that has a trailing space at the end of the heading. 
2. Select content in the source note (I had multiple lines selected, though I'm not sure that matters). 
3. Invoke the Filer plugin to move selected paragraphs
4. Select the target note
5. Pick the heading with the trailing space and hit enter. 

**Expected behavior**
Content was moved from the source note to the target note under the selected heading. If an error is encountered, content remains in source note and there is no data loss. 

**Screenshots**

**System**
 - Device: Mac
 - OS: macOS 12 (Monterey)
 - NotePlan Version: 3.7
 - Plugin Name & Version: "Filer" v1.0.0-beta

**Additional context**

**Plugin Console Log**
```
2022-10-27 14:08:13 | DEBUG | userInput / chooseHeading :: User picked existing heading #7 ('Mentoring for Employee') from 9 ..
2022-10-27 14:08:13 | DEBUG | jgclark.Filer v1.0.0-beta1 :: - Moving to note 'Tester' under heading: 'Mentoring for Employee'
2022-10-27 14:08:13 | DEBUG | jgclark.Filer v1.0.0-beta1 :: -> Inserting at start of section 'Mentoring for Employee'
Error: Couldn't find heading, this is case sensitive. Please check again.
2022-10-27 14:08:13 | DEBUG | jgclark.Filer v1.0.0-beta1 :: - Removing 4 paras from original note
```

**Comment by jgclark** [comment]
@edgauthier thanks for the clear report. I've not looked at the code yet, but I expect its a simple thing to fix.

**Comment by jgclark** [comment]
Fixed with commit cf2abaeb66897ec2e291321ab51625a60eb0e055 released as 1.0.0-beta2.
